inputs/templates/test/JoinRawCalls/JoinRawCalls.json.tmpl

```diff
@@ -15,6 +15,8 @@
   "JoinRawCalls.clustered_melt_vcfs" : [{{ test_batch.merged_melt_vcf | tojson }}],
   "JoinRawCalls.clustered_melt_vcf_indexes" : [{{ test_batch.merged_melt_vcf_index | tojson }}],
 
+  "JoinRawCalls.FormatVcfForGatk.formatter_args": "--fix-end",
```

Comment:
Does it also need `--scale-down-gq` since these are the raw calls that haven't had ends/GQ fixed in CleanVcf? That's what I see in the filtering instructions document as well

Reply:
No these are vcfs from `ClusterBatch` so there are no GQs